Automattic/jetpack
# Lazy Images: Hide lazy images when no js

Fixes #9281 by hiding lazy images when JS isn't turned on. Previously, a large blank spot would appear above the image which was caused by the lazy loaded image having the blank placeholder and then actual image being loaded via the noscript tag.

To test:

- Add lazy images to a post
- Load post with JS on
- Ensure lazy images load once
- Turn off JavaScript
- Reload post
- Ensure that the image loads via the noscript tag and the lazy images image is hidden. In other words, there shouldn't be a large blank spot
